packages/mobile/src/components/image/FastImage.tsx

```diff
@@ -15,12 +15,13 @@ export type ImageProps = Omit<FastImageProps, 'source'>
 export const FastImage = (props: FastImageProps) => {
   const { source, priority, ...other } = props
 
-  const imageSource =
-    typeof source === 'number'
-      ? source
-      : Array.isArray(source)
-      ? { uri: source[0].uri, priority }
-      : { uri: source.uri, priority }
+  const imageSource = !source
```

Comment:
nice, I think this fixes https://linear.app/audius/issue/PAY-1428/mobile-error-searching-for-ed-in-search-users-modal !